sys-kernel/kpatch/kpatch-0.4.0.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit linux-info linux-mod flag-o-matic
+
+if [[ "${PV}" == "9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/dynup/${PN}.git"
+else
+	SRC_URI="https://github.com/dynup/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+DESCRIPTION="Dynamic kernel patching for Linux"
+HOMEPAGE="https://github.com/dynup/kpatch"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="test examples +modules"
+
+DEPEND="
+	sys-devel/bison
+	dev-libs/elfutils
+	>sys-devel/gcc-4.9.3
```

Comment:
Compiler dependencies in DEPEND are meaningless due to gcc-config. You need to use tc-* functions.